README.md

```diff
@@ -195,7 +195,8 @@ const { flags, values, positionals } = parseArgs(argv, options);
 - Does the API specify whether a `--` was present/relevant?
   - no
 - Is `-foo` the same as `--foo`?
-  - no, `-foo` is a short option or options (WIP: https://github.com/pkgjs/parseargs/issues/2)
+  - no, `-foo` is a short option or options, with expansion logic that follows the
+    [Utility Syntax Guidelines in POSIX.1-2017](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html). `--foo bar`, `withValue: ['foo']`, expands to `-f`, `-o`, `-o bar`.
```

Comment:
The example expansion code is wrong... But in any case, I suggest leaving out the `withValue` case for the more likely flag usage. And likewise could avoid the double letter too. Keep it simple. I'll try adding some patch suggestions...

Reply:
```suggestion
    [Utility Syntax Guidelines in POSIX.1-2017](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html). `-bar` expands to `-b`, `-a`, `-r`.
```